src/Castle.Windsor/MicroKernel/IKernel.cs

```diff
@@ -148,6 +148,13 @@ IKernel AddFacility<T>(Action<T> onCreate)
 		/// <returns></returns>
 		IHandler GetHandler(String name);
 
+		/// <summary>
+		///   Returns the <see cref = "IHandler" />
+		///   for the all components.
+		/// </summary>
+		/// <returns>Handlers which implement sub dependency resolvers</returns>
+		IHandler[] GetHandlers();
```

Comment:
@jnm2 - I have made getting all handlers easy. This is going to start playing into validation of configurations if @jonorossi approves this PR.

Reply:
I needed to validate against "framework configuration components" accidentally registered in the container here: https://github.com/castleproject/Windsor/pull/376/commits/463da9f141e8a0fa8c87d066679ae5e028995796